test/cargo_build_script/tools_exec.rs

```diff
@@ -1,9 +1,9 @@
+
 #[test]
 pub fn test_tool_exec() {
     let tool_path = env!("TOOL_PATH");
     assert!(
         tool_path.contains("-exec-"),
-        "tool_path did not contain '-exec-': {}",
-        tool_path
+        "tool_path did not contain '-exec-'",
```

Comment:
I believe this change is related to https://github.com/rust-lang/rust-clippy/pull/10055

Unfortunately `#[allow(clippy::uninlined_format_args]` cannot be used here without requiring we raise our min supported Rust version above 1.59.0